portafly/src/i18n/locales/en/product.yml

```diff
@@ -0,0 +1,30 @@
+---
+# ------------------ Product > Create product
+# Page details
+
+page_title: Create product #--- PROPOSED
+page_title_description: Page title to create a product
+body_title: Create a new product #--- PROPOSED
```

Comment:
@porueesq shall it maybe say an "API product"?
We need to agree on a way to call _products_ across the entire UI and Docs

Reply:
The agreement was to call _Products_. See conversation in: https://3scale.slack.com/archives/C025Y9VUU/p1595411971272700